java/runtime/src/main/java/io/ray/runtime/utils/parallelactor/strategy/AlwaysFirstStrategy.java

```diff
@@ -0,0 +1,28 @@
+package io.ray.runtime.utils.parallelactor.strategy;
+
+import io.ray.api.parallelactor.ParallelStrategy;
+import io.ray.api.parallelactor.ParallelStrategyInterface;
+
+public class AlwaysFirstStrategy implements ParallelStrategyInterface {
```

Comment:
This strategy looks weird. In which case would someone want to always call the first instance? If so, why bother creating multiple instances?
BTW, a random strategy would make sense.